tests/unit/s2n_safety_test.c

```diff
@@ -335,6 +335,19 @@ int main(int argc, char **argv)
     CHECK_OVF(s2n_mul_overflow, uint32_t, 0x1FFFF, 0x1FFFF);
     CHECK_OVF(s2n_mul_overflow, uint32_t, ~0u, ~0u);
 
+    uint32_t result = 1;
+    EXPECT_SUCCESS(s2n_align_to(0,10,&result));
+    EXPECT_EQUAL(result, 0);
+
+    EXPECT_FAILURE(s2n_align_to(1,0,&result));
+
+    EXPECT_SUCCESS(s2n_align_to(10,16,&result));
+    EXPECT_EQUAL(result, 16);
+
+    EXPECT_SUCCESS(s2n_align_to(20,16,&result));
+    EXPECT_EQUAL(result, 32);
+
+    EXPECT_FAILURE(s2n_align_to(UINT32_MAX, 4, &result));
```

Comment:
Would it be worth adding a test case that uses page_size? since that's how we use it for real.

Reply:
Makes sense.

Reply:
Done